packages/legacy/src/queries/providers.ts

```diff
@@ -47,30 +45,27 @@ export const useClusterProvidersQuery = (
   namespace: string
 ): UseQueryResult<IKubeList<IProviderObject>> => {
   const providerResource = createResource(ForkliftResourceKind.Provider, namespace);
-  return useMockableQuery<IKubeList<IProviderObject>>(
-    {
-      queryKey: 'cluster-providers',
-      queryFn: async () => await consoleFetchJSON(providerResource.listPath()),
-      refetchInterval: usePollingContext().refetchInterval,
-    },
-    mockKubeList(MOCK_CLUSTER_PROVIDERS, 'Providers')
-  );
+
+  const result = useQuery<IKubeList<IProviderObject>, unknown, IKubeList<IProviderObject>>({
```

Comment:
i'm missing something, the idea was to not change the query code ? what am i missing ?